CriticalMass/MapInfoViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  CriticalMaps
+
+import CoreLocation
+import UIKit
+
+// https://material.io/design/motion/speed.html#easing
+extension UICubicTimingParameters {
```

Comment:
Should this be a private extension which is only used in this file or do we possibly need it somewhere else? When we possibly need it somewhere else, I would prefer to put this in an own file.